File: generator/output/entityModel/InstanceImageModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.lang.Boolean;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* InstanceImage
*/
@Data()
public class InstanceImageModel {
  public Long id = null;

  /**
  *
  */
  public Integer deleted = null;

  /**
  * 忙碌 false为可用
  */
  public Boolean busy = null;

  /**
  *
  */
  public Long application_id = null;

  /**
  * 镜像 ID
  */
  public String image_id = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/instance-image.model.ts
class InstanceImageModel {
  public id: number = 0;

  // deleted 
  public deleted: boolean = false;

  // busy 忙碌 false为可用
  public busy: boolean = false;

  // application_id 
  public application_id: number = 0;

  // image_id 镜像 ID
  public image_id: string;

  public created_at: string;

  public updated_at: string;
}

const InstanceImageModelFields = {
  id: "id",
  deleted: "deleted",
  busy: "busy",
  application_id: "application_id",
  image_id: "image_id",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {InstanceImageModel, InstanceImageModelFields};
